Smt/Reconstruct/Real/TransFns/ArithTransPi.lean

```diff
@@ -0,0 +1,140 @@
+/-
+Copyright (c) 2021-2023 by the authors listed in the file AUTHORS and their
+institutional affiliations. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Tomaz Mascarenhas
+-/
+
+/-
+Implementation of:
+https://cvc5.github.io/docs/cvc5-1.0.2/proofs/proof_rules.html#_CPPv4N4cvc58internal6PfRule14ARITH_TRANS_PIE
+-/
+
+import Lean
+import Qq
+import Mathlib.Analysis.SpecialFunctions.Trigonometric.Basic
+import Mathlib.Analysis.Real.Pi.Bounds
+
+import Smt.Reconstruct.Util
+import Smt.Reconstruct.Real.Polynorm
```

Comment:
Polynorm is not used here. Please remove this import and add it back to `Smt/Reconstruct/Real`.

Reply:
Hm I'm using `Real.normNum` in this file; should I redefine it there? Or maybe move this function to one of the `utils` files?

Reply:
Good point... forgot about `Real.normNum`.